src/style.css

```diff
@@ -0,0 +1,60 @@
+* {
+  font-family: "Roboto", sans-serif;
+  color: #000;
+  background-color: #fff;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  padding: 23px 0;
+}
+
+.main-header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  display: flex;
+  height: 60px;
+  justify-content: space-between;
+  align-items: center;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+}
+
+.nav-item {
+  margin-right: 20px;
+  display: flex;
+  align-items: center;
+}
+
+.nav-item:last-child {
+  margin-right: 0;
+}
+
+.nav-item a:hover {
+  color: #00acdc;
+}
+
+.is-active {
```

Comment:
Suggestion: in big projects, you will have many items which can be `active`. Better to define which element exatly you want to style with class `is-active`, because this class is like `utility class`.
```css
.nav-item a.is-active {...}
```